components/automate-deployment/pkg/server/startup_helpers.go

```diff
@@ -37,6 +38,35 @@ func habSupVersion() string {
 	return strings.TrimSpace(output)
 }
 
+// maybeResetHabPath adds the bindir from our manifest's hab package
+// to our PATH. This is a developent-mode only operation to account
+// for test cases where we want to deploy with the latest Habitat.
+func maybeResetHabPath(s *server) {
```

Comment:
A perhaps better path would be to _always_ use the version from the manifest via `hab pkg exec`, avoiding this problem completely; however, that change will require a bit more testing.

Reply:
i'm kind of surprised we're not already automatically calling the hab from the manifest